plutus-ir/test/datatypes/listMatch.plc

```diff
@@ -0,0 +1,22 @@
+(let (rec)
```

Comment:
Kind of a nit, and I've been lazy about this, but it would be nice if the PIR files had `.pir` in their extension instead of `.plc`.

Reply:
Very much agree with this.